### PR TITLE
[INJIWEB-723] : release version upgrade with postman and stoplight inclusion

### PR DESCRIPTION
**Pull request by vijay151096**

### CODE REVIEW

File: docs/postman-collections/MIMOTO.postman_collection.json [hunk 1]
```diff
@@ -23,7 +32,150 @@
...
 						"method": "GET",
 						"header": [],
-						"url": "{{HOST}}/issuers/ESignet"
+						"url": {
+							"raw": "{{HOST}}/issuers/ESignet",
+							"host": [
+								"{{HOST}}"
+							],
+							"path": [
+								"issuers",
```
Comment (swatigoel) [inline docs/postman-collections/MIMOTO.postman_collection.json:41]: We don't have ESignet issuer now

File: docs/postman-collections/MIMOTO.postman_collection.json [hunk 2]
```diff
@@ -14,7 +14,16 @@
 					"request": {
 						"method": "GET",
 						"header": [],
-						"url": "{{HOST}}/issuers/"
+						"url": {
+							"raw": "{{HOST}}/issuers/",
```
Comment (swatigoel) [inline docs/postman-collections/MIMOTO.postman_collection.json:18]: remove `/` after issuers

**Review by swatigoel — CHANGES_REQUESTED** [review]

File: docs/postman-collections/MIMOTO.postman_collection.json (hunk 1 above)
Comment (vijay151096) [reply docs/postman-collections/MIMOTO.postman_collection.json:41]: Fixed

File: docs/postman-collections/MIMOTO.postman_collection.json (hunk 2 above)
Comment (vijay151096) [reply docs/postman-collections/MIMOTO.postman_collection.json]: removed

File: pom.xml [hunk 3]
```diff
@@ -33,11 +33,11 @@
...
         <!-- mosip -->
-        <mosip.kernel.core.version>1.2.1-SNAPSHOT</mosip.kernel.core.version>
-        <mosip.biometric.util.version>1.2.1-SNAPSHOT</mosip.biometric.util.version>
-        <mosip.kernel.auth.adapter.version>1.2.1-SNAPSHOT</mosip.kernel.auth.adapter.version>
-        <kernel.biometrics.api.version>1.2.1-SNAPSHOT</kernel.biometrics.api.version>
-        <kernel.cbeffutil.version>1.2.1-SNAPSHOT</kernel.cbeffutil.version>
+        <mosip.kernel.core.version>1.3.0-beta.1</mosip.kernel.core.version>
+        <mosip.biometric.util.version>1.3.0-SNAPSHOT</mosip.biometric.util.version>
+        <mosip.kernel.auth.adapter.version>1.3.0-SNAPSHOT</mosip.kernel.auth.adapter.version>
+        <kernel.biometrics.api.version>1.3.0-SNAPSHOT</kernel.biometrics.api.version>
```
Comment (swatigoel) [inline pom.xml:39]: version should be 1.3.0-beta.1 as it's part of commons

**Review by swatigoel — CHANGES_REQUESTED** [review]

**Review by swatigoel — APPROVED** [review]

**Pull request merged**